BungieFrameWork/BFW_Headers/BFW_Particle3.h

```diff
@@ -944,9 +944,9 @@ extern const P3tString P3gEventName[P3cMaxNumEvents];
 
 #define P3cLensFrames_TypeOffset	16
 #define P3cLensFrames_TypeBits		16
-#define P3cLensFrames_TypeMask		(((1 << P3cLensFrames_TypeBits) - 1) << P3cLensFrames_TypeOffset)
+#define P3cLensFrames_TypeMask		((((UUtUns32)1 << P3cLensFrames_TypeBits) - 1) << P3cLensFrames_TypeOffset)
```

Comment:
How about using a `U` suffix?
```suggestion
#define P3cLensFrames_TypeMask		(((1U << P3cLensFrames_TypeBits) - 1) << P3cLensFrames_TypeOffset)
```

Reply:
That works. Change adopted.